apps/web/next.config.mjs

```diff
@@ -23,7 +23,7 @@ const profileTarget = {
 };
 
 const getRedirectConfig = profile => {
-  const target = profileTarget[profile || 'dev'] || profileTarget['dev'];
+  const target = profileTarget[profile || 'dev'];
```

Comment:
```ts
const target = profileTarget[profile || 'dev'] || profileTarget['dev'];
```

when profile is `undefined` will fallback to `'dev'`, `profileTarget[profile || 'dev']` has been achieved.

Reply:
if we pass profile like `abcd`, at this point target is `undefined`

Reply:
oh, got it~